pkg/ruleengine/v1/helpers.go

```diff
@@ -153,3 +153,32 @@ func interfaceToStringSlice(val interface{}) ([]string, bool) {
 	}
 	return nil, false
 }
+
+func CompareDynamic(dynamicPath, regularPath string) (string, bool) {
+	const dynamic = "<dynamic>"
```

Comment:
this should use the same const as the storage, otherwise we might get into trouble

Reply:
DONE